src/mscorlib/src/System/Collections/Comparer.cs

```diff
@@ -41,18 +41,6 @@ public sealed class Comparer : IComparer , ISerializable
             Contract.EndContractBlock();
             m_compareInfo = culture.CompareInfo;
         }
-        
-        private Comparer(SerializationInfo info, StreamingContext context) {            
```

Comment:
This whole file should be deleted. The shipping public implementation is in corefx. It is just that something is keeping alive in CoreLib.